pkg/koordlet/statesinformer/impl/states_noderesourcetopology.go

```diff
@@ -100,8 +98,8 @@ func (n *nodeTopologyStatus) updateNRT(nrt *v1alpha1.NodeResourceTopology) {
 
 	nrt.TopologyPolicies = []string{string(n.TopologyPolicy)}
 
-	// TBD: merge with the existing
-	nrt.Zones = n.Zones
+	// trim useless zone name and merge with the existing zone list
+	nrt.Zones = util.MergeZoneList(util.TrimZoneList(nrt.Zones, "fake-name"), n.Zones)
```

Comment:
is the `fake-name` still necessary?

Reply:
Currently, the NRT is created firstly with no actual zone resources and updated later. It still keeps the `fake-name` in creation to avoid breaking up other components like the scheduler's assumption.

Reply:
Removed. There was no at least one zone assumption so the `fake-name` makes nonsense.